contracts/StablecoinConverter.sol

```diff
@@ -248,17 +266,20 @@ contract StablecoinConverter is EpochTokenLocker {
         uint128 executedBuyAmount,
         uint128 buyTokenPrice,
         uint128 sellTokenPrice
+        // uint128 feeDenominator
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2FFGTVmzksb2j0k%2Fgiphy.gif&token=oops" data-gitmeme-token="oops"><img src="https://media1.giphy.com/media/FGTVmzksb2j0k/giphy.gif" title="Created by gitme.me with /oops"/></a>

Reply:
Damn it!